tests/test_tokenization_mbart.py

```diff
@@ -39,6 +39,7 @@ class MBartTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
     tokenizer_class = MBartTokenizer
     rust_tokenizer_class = MBartTokenizerFast
     test_rust_tokenizer = True
+    test_sentencepiece = True
```

Comment:
`mbart` was not flagged as a sentencepiece tokenizer in the tests until now :slightly_smiling_face: